infra/dockerfile.commit_retrieval

```diff
@@ -2,18 +2,22 @@ FROM mozilla/bugbug-base:latest
 
 COPY infra/git_hyper_blame_patch /root/git_hyper_blame_patch
 
-# Mercurial need Python2 :(
+# Setup mercurial from its own website, without install pip2 which has a lot of dependencies
+ARG MERCURIAL_VERSION="5.1"
+
 # git and git-hyper-blame (from depot_tools) are required by the annotate pipeline.
 RUN apt-get update && \
-    apt-get install -y python python-pip git curl && \
-    python2 -m pip install --disable-pip-version-check --no-cache-dir mercurial==5.1 && \
+    apt-get install --no-install-recommends -y python python-dev python-docutils build-essential git curl && \
```

Comment:
build-essential has many dependencies, can't you install just gcc instead like https://github.com/mozilla/code-coverage/blob/master/bot/ci/bootstrap.sh does?

Reply:
Ok, will do that in the next edit.